scripts/postlink/ios/postlink.js

```diff
@@ -129,11 +156,13 @@ function getPlistPath(){
     }
 
     var INFO_PLIST_PROJECT_KEY = 'INFOPLIST_FILE';
-    var plistPathValue = parsedXCodeProj.getBuildProperty(INFO_PLIST_PROJECT_KEY);
+    var targetProductName = package ? package.name : null;
+
+    var plistPathValue = getBuildSettingsPropertyMatchingTargetProductName(parsedXCodeProj, INFO_PLIST_PROJECT_KEY, targetProductName);
```

Comment:
Hmm, I believe we want to use 'Release' build and indeed if we pass the 4th parameter to `getBuildSettingsPropertyMatchingTargetProductName` function as 'Release' it will take the only release build if it exists. 
I'm not sure if pbxproj file always have the same order of debug/release sections as in the example? If yes - then 'Release' will always be the last one and correctly picked up. But when we invoke function with only three params it will work if we have only one definition for build (I mean release or debug). Is it a use case?
@silhouettes, Please let me know if you think it doesn't make sense and we can always expect to get 'release' build.